app/helpers/jwt_helper.rb

```diff
@@ -5,11 +5,11 @@ def build_jwt(user, valid_for_hours = 48)
     exp = Time.now.to_i + (valid_for_hours * 60 * 60)
     payload = { expiration: exp,
                 user: user }
-    JWT.encode(payload, ENV['JWT_SECRET'], 'HS256')
+    JWT.encode(payload, ENV['JWT_SECRET_KEY'], 'HS256')
   end
 
   def decode_jwt(jwt)
-    decoded_token = JWT.decode(jwt, ENV['JWT_SECRET'], true)
+    decoded_token = JWT.decode(jwt, ENV['JWT_SECRET_KEY'], true)
```

Comment:
JWT tokens are supposed to be generated before making api calls. Hence this key doesn't need to be in **kubernetes_config**.
* Generate the key before running this script. Export the `JWT_SECRET_KEY` in bash which will be read in the `ENV` here.
* Ask David Kronenberg on how to generate the key. It must be documented somewhere. If it is not, mention it on slack.
* Revert the kubernetes_openhub commit which added the jwt secret key.

Reply:
Discard this. We do need *JWT_SECRET_KEY* as an environment value. This is the secret key used to sign new JWT tokens.